src/EnergyPlus/DataWater.hh

```diff
@@ -317,6 +329,9 @@ struct DataWaterData : BaseGlobalStruct
     bool AnyWaterSystemsInModel = false;    // control flag set true if any water systems
     bool WaterSystemGetInputCalled = false; // set true once input data gotten.
     bool AnyIrrigationInModel = false;      // control flag set true if irrigation input for ecoroof DJS PSU Dec 2006
+    bool UsePrecipitation = false;          // whether to include the rain fall amount in the actual irrigation amount
```

Comment:
This is not used now.

Reply:
Right, I forgot to modify this header file